README.md

```diff
@@ -78,35 +78,34 @@ Every few weeks, we release a new Zebra beta [release](https://github.com/ZcashF
 Zebra's network stack is interoperable with `zcashd`,
 and Zebra implements all the features required to reach Zcash network consensus.
 
-The goals of the beta release series are for Zebra to act as a fully validating Canopy and NU5 node.
+The goals of the beta release series are for Zebra to act as a fully validating Zcash node for 
+all applicable consensus rules as of NU5 activation.
+
 Currently, Zebra does not validate the following Zcash consensus rules:
 
 #### NU5
 - ZIP-155 - Parse addrv2 in Zebra
 - Full validation of Orchard transactions from NU5 onwards
    - Check that at least one of enableSpendsOrchard or enableOutputsOrchard is set
-   - Validation of Orchard anchors
+   - Validation of Orchard anchors (root of the Orchard note commitment tree)
    - Validation of Halo2 proofs
-   - Validation of orchard note commitment trees
 
 #### NU4 - Canopy
 - Calculation of Block Subsidy and Funding streams
 - Validation of coinbase miner subsidy and miner fees
 - Validation of shielded outputs for coinbase transactions (ZIP-212/ZIP-213)
 
 #### NU1 - Sapling
-- Validation of Sapling anchors
-- Validation of sapling note commitment trees
-- Validation of JoinSplit proofs using Groth16 verifier
+- Validation of Sapling anchors (root of the Sapling note commitment tree)
+- Validation of Sprout JoinSplit-on-Groth16 proofs
 
 #### NU0 - Overwinter
 - ZIP-203: Transaction Expiry consensus rules
 
 #### Sprout
-- Validation of Sprout anchors
-- Validation of JoinSplit proofs using BCTV14 verifier
```

Comment:
Note: this deletion is correct, we don't intend to validate BCTV14 proofs, because they were only used before the Sapling network upgrade, and Zebra checkpoints on Canopy.